web/channels/player_channel.ex

```diff
@@ -34,7 +34,12 @@ defmodule Yggdrasil.PlayerChannel do
   end
 
   def handle_in("player_cmd", %{ "text" => text }, socket) do
-    Player.run_cmd(socket.assigns.user, Message.command(text))
-    {:noreply, socket}
+    case Parser.parse(text) do
+      {:ok, cmd} ->
+        Player.run_cmd(socket.assigns.user, cmd)
+        {:noreply, socket}
+      {:error, reason} ->
+        {:reply, {:error, Message.error(reason)},  socket}
```

Comment:
I don't think we want to `{:reply, {:error, ..}}` here. Client doesn't see the message in particular the reason I'm against this is how the message is manifested on the web socket. 

``` json
/*bogus command */
{"topic":"player:1","ref":"8",
 "payload":{"status":"error","response":{"type":"error","message":"Unknown command.\n"}},
 "event":"phx_reply"}
 /* in particular event type "phy_reply" doesn't seem productive for catching game errors */

/* good command*/
{"topic":"player:1","ref":null,
 "payload":{"type":"info","message":"You head towards east"},
 "event":"event"}
```
While the parse failed, which is an error I think it may be more beneficial to either manifest it back on the 'event' channel and have on place to listen to it, as it's game response (there will be many you can't do that's I'm sure) or maybe we do something like

```elixir
{:error, reason} ->
  # wrap in a payload if need be just a quick example (can't remember the format of Messages...)
  broadcast! socket, "error", Message.error(reason) 
```
This give a better point to latch onto for errors, because the phy_reply subject has several message on it including a heartbeat, so if we latched on to it we would have to constantly filter out that heart beat and ensure it was "game" error we are sending back.